packages/richtext-lexical/src/features/toolbars/inline/client/Toolbar/index.tsx

```diff
@@ -105,6 +106,7 @@ function ToolbarGroupComponent({
           <ToolbarDropdown
             anchorElem={anchorElem}
             editor={editor}
+            group={group}
             groupKey={group.key}
```

Comment:
Could you remove group.key and group.items, since we're passing the entire group anyways?